docs/index.md

```diff
@@ -28,7 +28,7 @@ This tool is intended for use by developers who are experienced with JavaScript.
 ## Including the Buy SDK
 
 ```html
-<script src="http://sdks.shopifycdn.com/js-buy-sdk/latest/shopify-buy.polyfilled.globals.min.js"></script>
+<script src="http://sdks.shopifycdn.com/js-buy-sdk/v0/latest/shopify-buy.umd.polyfilled.min.js"></script>
```

Comment:
It's a normal thing with semantic versioning. It prevents us from pushing breaking API changes out to dozens or hundreds of apps and destroying other people's products. Since we _are_ using sem-ver, it definitely fixes part of the problem. We can't actually change the functional spec of the library without bumping the major version. We can however optimize and fix bugs, and progressively enhance as we go, and people pointing to `latest` will still get the benefit of that.